ui/pages/Search/components/filters/ProjectFamiliesFilter.jsx

```diff
@@ -27,15 +38,28 @@ class ProjectFamiliesFilterInput extends React.PureComponent {
     onChange: PropTypes.func,
   }
 
-  allFamiliesSelected = () => {
+  multiFamiliesSelected = () => {
     const { familyOptions, value } = this.props
-    return !value.familyGuids || value.familyGuids.length === familyOptions.length
+    if (!value.familyGuids || value.familyGuids.length === familyOptions.length) {
+      return ALL
+    }
+    if (this.unsolvedFamilyGuids().sort().join(',') === value.familyGuids.sort().join(',')) {
+      return UNSOLVED
+    }
+    return null
+  }
+
+  unsolvedFamilyGuids = () => {
+    const { familyOptions } = this.props
+    return familyOptions.filter(
+      ({ analysisStatus }) => !SOLVED_FAMILY_STATUS_OPTIONS.has(analysisStatus),
+    ).map((({ value }) => value))
   }
```

Comment:
It is neater for the function takes in the `familyOptions` parameter rather than get it from `this.props`. And it can also return a sorted and comma-joined string.

Reply:
That threw an error with our linter, I tried that. And it can't return a string, the value of the field that is sent to the server is a list